digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala

```diff
@@ -310,14 +311,14 @@ class RoadAddressServiceSpec extends FunSuite with Matchers{
       val addressList = RoadAddressDAO.fetchByLinkId(Set(linkId))
       addressList should have size (1)
       val address = addressList.head
-      address.floating should be (false)
+      address.isFloating should be (false)
       address.geometry shouldNot be (roadLink.geometry)
       roadAddressService.checkRoadAddressFloatingWithoutTX(Set(address.id))
       dynamicSession.rollback()
       val addressUpdated = RoadAddressDAO.queryById(Set(address.id)).head
       addressUpdated.geometry shouldNot be (address.geometry)
       addressUpdated.geometry should be(roadLink.geometry)
-      addressUpdated.floating should be (false)
+      addressUpdated.isFloating should be (true)
```

Comment:
Is this okay that I changed the test and assumed that the implementation was correct?

Reply:
yep i guess somehow the vvh changed for this link making it floating? At least the changes didnt deal with some new logic to detect floatings. Is it possible that in future the test fails again due to diff vvh geometry for this link?

Reply:
If this is using real VVH, then yes, test can fail in the future. We should change all the tests to use mock VVH so that we can run the tests independently.

Reply:
But not doing this mock VVH change in this pull request.